backend/src/main/resources/db/migration/V2023.01.10.1147__create_user.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE spar.user (
```

Comment:
I would highly suggest to always use `IF NOT EXISTS` to every SQL statement to prevent issues during migration.